tests/kernel/nvmftests.pm

```diff
@@ -28,7 +28,7 @@ sub run {
 
     assert_script_run('cd /var/lib/nvmftests');
     assert_script_run('ln -sf tests/config config');
-    assert_script_run('nose2 --verbose', 1200);
+    assert_script_run('nose2 --verbose', 3600);
```

Comment:
that is … long . Are tests this long really useful? Maybe the virtualization is slowing things down here too much?

Reply:
Yes, the test is useful. And no, virtualization is not the problem.

Reply:
Do we have an idea why it failed? With 3600 we are closer to the general timeout of the test suite which is 2 hours.

Reply:
A typical test run on a local VM takes  ~10 minutes. So 20 minutes simply seem to be too short for an environment with limited CPU bandwith.

Reply:
Alright, let's give it a shot. In case it doesn't help we should investigate deeper and find the root cause, (e.g. performance degradation).